lib/app-components/addon/components/error-page/styles.scss

```diff
@@ -9,3 +9,25 @@
         width: 100%;
     }
 }
+
+.button {
```

Comment:
I think the ultimate solution would be to add button styling options to osf-link, but I'm not keen to do that here. Since this is a minor thing, I think I'll let this one go for now if that's all right.